src/DynamoCoreWpf/UI/GuidedTour/Tooltip.cs

```diff
@@ -102,6 +107,17 @@ private void DrawPointerDirection(PointerDirection direction)
                 pointX3 = realWidth;
                 pointY3 = Height - PointerHeight / 2 - PointerVerticalOffset;
             }
+            else if (direction == PointerDirection.BOTTOM_DOWN)
+            {
+                pointX1 = PointerHorizontalOffset;
+                pointY1 = Height + PointerDownHeight;
+
+                pointX2 = PointerDownWidth + PointerHorizontalOffset;
+                pointY2 = Height + PointerDownHeight;
+
+                pointX3 = PointerDownWidth / 2 + PointerHorizontalOffset;
+                pointY3 = realHeight - PointerHeight;
```

Comment:
HI @filipeotero Without the calculation, where the down pointer  would be?

Reply:
Hi @QilongTang, the default pointer is TOP_LEFT. 

![image](https://user-images.githubusercontent.com/89042471/142000863-c431a86f-a0ba-46c1-98d7-1887625e351e.png)